crates/biome_service/src/file_handlers/mod.rs

```diff
@@ -469,6 +472,20 @@ pub(crate) struct FormatterCapabilities {
     pub(crate) format_on_type: Option<FormatOnType>,
 }
 
+type Search = fn(
+    &BiomePath,
+    &DocumentFileSource,
+    AnyParse,
+    &GritQuery,
+    WorkspaceSettingsHandle,
```

Comment:
nit: Considering the great number of arguments, we could evaluate a `struct` instead